### PR TITLE
TNO-1150: Add stories to evening overview

### PR DESCRIPTION
**Pull request by jdtoombs**
In this PR:

- user may now add related stories to the av overview
- each section will filter out related content based on the show/program
- new filter option for seriesId

link-content

### CODE REVIEW

File: app/editor/src/features/reports/av-overview/OverviewSection.tsx [hunk 1]
```diff
@@ -58,7 +58,7 @@ export const OverviewSection: React.FC<IOverviewSectionProps> = ({ editable = tr
         editable={editable}
         open={!section.sourceId && !section.otherSource && !section.seriesId}
       />
-      <OverviewGrid index={index} editable={editable} />
+      <OverviewGrid index={index} editable={editable} seriesId={section.seriesId} />
```
Comment (Fosol) [inline app/editor/src/features/reports/av-overview/OverviewSection.tsx:61]: You don't need to pass in this value, it's already part of the FormikContext

File: app/editor/src/features/reports/av-overview/OverviewGrid.tsx [hunk 2]
```diff
@@ -10,24 +12,52 @@ import {
...
+  /** helps to filter out content related to the show/program */
+  seriesId?: number;
 }
 
 /** OverviewGrid contains the table of items displayed for each overview section. */
-export const OverviewGrid: React.FC<IOverviewGridProps> = ({ editable = true, index }) => {
+export const OverviewGrid: React.FC<IOverviewGridProps> = ({
+  editable = true,
+  index,
+  seriesId,
```
Comment (Fosol) [inline app/editor/src/features/reports/av-overview/OverviewGrid.tsx:35]: You can get this value more reliably from `items.sectionId`.

File: app/editor/src/features/reports/av-overview/OverviewGrid.tsx [hunk 3]
```diff
@@ -10,24 +12,52 @@ import {
...
   const items = values.sections[index].items;
 
+  /** fetch pieces of content that are related to the series to display as options for associated clips */
+  React.useEffect(() => {
+    findContent({
+      seriesId: seriesId,
+      contentTypes: [],
+    }).then((data) =>
+      setClips(
+        data.items.map((c) => ({
```
Comment (Fosol) [inline app/editor/src/features/reports/av-overview/OverviewGrid.tsx:51]: You can use thew `new OptionItem(...)` class instead of manually casting.

**Review by Fosol — CHANGES_REQUESTED** [review]

File: app/editor/src/features/reports/av-overview/OverviewGrid.tsx (hunk 3 above)
Comment (jdtoombs) [reply app/editor/src/features/reports/av-overview/OverviewGrid.tsx:51]: Right! Will change all that up, also realized as I was falling asleep last night that I also need to filter by the date as well

**Comment by Fosol** [comment]
You will also need to update to the latest version `tno-core:0.0.309` and then republish the next version `0.0.310`

**Review by Fosol — APPROVED** [review]

**Pull request merged**